index.js

```diff
@@ -0,0 +1,116 @@
+const menuBtn = document.querySelector('.menu-bars');
+
+function openMenu() {
+  document.querySelector('.mobile-menu').style.display = 'block';
+}
+
+menuBtn.addEventListener('click', openMenu);
+
+const closeBtn = document.querySelector('.menu-x');
+
+function closeMenu() {
+  document.querySelector('.mobile-menu').style.display = 'none';
+}
+
+closeBtn.addEventListener('click', closeMenu);
+
+const guests = [
+  {
+    image: './assets/guests/guest1.png',
+    name: 'FRTT',
+    info: 'Streamer for Team Liquid.',
+    description: 'FRTT is a famous Valorant streamer on Twitch.',
+    id: '1',
+  },
+
+  {
+    image: './assets/guests/guest2.png',
+    name: 'Takayuki Nakayama',
+    info: 'Capcom',
+    description: 'Takauyuki is know for being the director and designer for the famous Street Fighter.',
+    id: '2',
+  },
+
+  {
+    image: './assets/guests/guest3.png',
+    name: 'Techzz',
+    info: 'TFT streamer.',
+    description: 'Techzz became famous doing TFT lives on Twitch.',
+    id: '3',
+  },
+
+  {
+    image: './assets/guests/guest4.png',
+    name: 'Takashi Iizuka',
+    info: 'Sega',
+    description: 'Takashi is the creative director for the game series Sonic the Hedgehog.',
+    id: '4',
+  },
+
+  {
+    image: './assets/guests/guest5.png',
+    name: 'Daiki',
+    info: 'Proplayer for Team Liquid.',
+    description: 'Daiki became famous after becoming a Valorant proplayer for Team Liquid.',
+    id: '5',
+  },
+
+  {
+    image: './assets/guests/guest6.png',
+    name: 'Yayah',
+    info: 'Streamer on Twitch',
+    description: 'Yayah is a famous streamer on Twitch.',
+    id: '6',
+  },
+];
+
+const guestSection = document.getElementById('guests-section');
+
+guests.forEach((item) => {
+  guestSection.innerHTML += ` 
```

Comment:
- [ ] I would suggest you, please use cards equally from both sides. The featured speaker has a margin from left and right and two columns is distributed equally from both side.
### Your design
![his](https://user-images.githubusercontent.com/81727988/191888273-87a47414-92b2-432b-888d-933a6ac3d49d.png)
### Microverse design
![figma](https://user-images.githubusercontent.com/81727988/191888327-ec57faf3-21e0-49ae-ac8c-ca74f8d68644.png)